packages/dataviews/src/layouts/table/index.tsx

```diff
@@ -238,7 +238,7 @@ function TableRow< Item >( {
 					onChangeSelection(
 						selection.includes( id )
 							? selection.filter( ( itemId ) => id !== itemId )
-							: [ ...selection, id ]
```

Comment:
Why do we need to clear the existing selection when the user clicks the row? This means that we'd have two affordances (checkbox & row click) that work differently:

https://github.com/user-attachments/assets/7a8e0f3a-bf49-4608-8445-fd6cc7c2467d

Reply:
yes. See https://github.com/WordPress/gutenberg/pull/63600#issuecomment-2232870658 and the following comments.

Reply:
I see this is intentional https://github.com/WordPress/gutenberg/pull/63600#issuecomment-2232870658

Reply:
I see where this is coming from: we want to optimize for the quick edit flow, otherwise it's quite cumbersome 👍